AutoMaxLair.py

```diff
@@ -417,15 +417,15 @@ def select_pokemon(inst) -> str:
                 take_pokemon = True
                 break
         elif inst.num_caught == 4 and 'ball saver' in inst.mode.lower():
-            if inst.check_sufficient_ore(inst.consecutive_resets + 2):
+            if inst.check_sufficient_ore(inst.consecutive_resets, 2):

```

Comment:
I don't think it's necessary to pass the consecutive resets as an argument—just use self.consecutive resets inside the function.